### DISCUSSION TITLE
LDAP Login when using domain

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3356] [Help / Questions]
**Describe the bug**
Can't et LDAP authentication to work:
`000004DC: LdapErr: DSID-0C090A4C, comment: In order to perform this operation a successful bind must be completed on the connection., data 0, v3839`

Have working details and ldap in php. 

**To Reproduce**
Setting up LDAP authentication. I used same thing as I have in my php script.
```php
//binding
$ldapbind = @ldap_bind($ldapconn, "EU\\". $username, $password);
$filter="(samaccountname=".$username.")";.$result = ldap_search($ldapconn,$ldaptree, $filter)
//options for search
ldap_set_option($ldapconn, LDAP_OPT_REFERRALS, 0);
ldap_set_option($ldapconn, LDAP_OPT_PROTOCOL_VERSION, 3);
```

**Expected behavior**
Login and getting username from LDAP

**Screenshot**
![image](https://user-images.githubusercontent.com/45201687/74336515-51bd1200-4d96-11ea-97f2-8167933bfa10.png)

**Host Info (please complete the following information):**
 - Windows Server

**Additional context**
It's weird, as I can't login with domain\\username in ldap browser, but that pattern only works in php.

**Comment by NGPixel** [comment]
You need to fill in the Admin Bind DN and Admin Bind Credentials fields.

**Comment by ghost** [comment]
But what are they? There is no admin on ldap server I am using. Or I don't understand those fields

**Comment by NGPixel** [comment]
It's the username and password used to connect to the LDAP server. It can be any user as long as they have the proper permissions in AD to lookup all users.

The username must be the DN of the user.

**Comment by ghost** [comment]
Ah, makes sense, kind of because all users can do it. But credential for non admin users ares till checked by username and password on login right?

**Comment by NGPixel** [comment]
Correct.

**Comment by gdevenyi** [comment]
Technically anonymous binding should also be an option if the server supports it.
